src/sentry/models/authenticator.py

```diff
@@ -289,9 +290,9 @@ def __init__(self, authenticator=None):
     def get_codes(self):
         rv = []
         if self.is_enrolled:
-            h = hmac.new(key=self.config["salt"].encode("utf-8"), msg=None, digestmod=hashlib.sha1)
+            h = hmac.new(key=force_bytes(self.config["salt"]), msg=None, digestmod=hashlib.sha1)
```

Comment:
We could maybe just use this directly, since this should always be the value generated in `generate_new_config`, which will be bytes. I guess using `force_bytes` here doesn't hurt though.